.pipelines/stages/jobs/nuget-win-packaging-job.yml

```diff
@@ -45,7 +45,7 @@ jobs:
     ${{ if eq(parameters.ep, 'cpu') }}:
       value: ''
     ${{ if eq(parameters.ep, 'cuda') }}:
-      value: 'Gpu'
+      value: '.Cuda'
```

Comment:
We will need to change how the nuspec dependencies are defined in the nuspec. See [this line](https://github.com/microsoft/onnxruntime-genai/blob/78df9444f71476d4a37e3ede0a55a2b2920adc41/nuget/Microsoft.ML.OnnxRuntimeGenAI.nuspec#L20) which shows the dependency on `Microsoft.ML.OnnxRuntime.Gpu`. It will now change to `Microsoft.ML.OnnxRuntime.Cuda` but this package does not exist. We will have to update the nuspec file (probably take in a new argument for onnxruntime package name)

Reply:
Do you meant `Microsoft.ML.OnnxRuntimeGenAI.Cuda`?

Reply:
I see what you meant.

Reply:
add the new variables called ort_nuget_ext to distiguish from genai_nuget_ext, also include a new property called ort_verson.